### ISSUE TITLE
Can't find dataset link for DOI 10.1017/S0960258518000211

### ISSUE THREAD

**Issue by afandian** [CrossRef/event-data-enquiries#186]
This probably should be indexed, but it's not present in the API.

**Comment by kjgarza** [comment]
It seems to be that in the Crossref API there is no information about the datasets DOIs. Maybe that is why EventData is not picking it up.

> http://api.crossref.org/works/10.1017/S0960258518000211

```
{
key: "S0960258518000211_ref16",
year: "1994",
volume-title: "Genebank Standards"
},
```

```
{
key: "S0960258518000211_ref15",
year: "2014",
volume-title: "Genebank Standards for Plant Genetic Resources for Food and Agriculture"
},
```

**Comment by jenniferlin15** [comment]
Exactly. Publisher did not deposit the DataCite DOI for the dataset in the reference, so Event Data cannot pick it up. Closing ticket.

**Comment by mfenner** [comment]
Looking at this again, we actually see 19 DataCite DOIs (prefix 10.18730) in the Crossref REST API, and they now also appear in Event Data, with the events added September 5. We can close the ticket as everything is in Event Data now.
